Developer-Guide.md

```diff
@@ -62,6 +63,16 @@ $ sudo kata-runtime kata-check
 
 If your system is *not* able to run Kata Containers, the previous command will error and explain why.
 
+## Disable initrd
+
+Unless you plan to use an initial ramdisk (`initrd` or `initramfs`):
+
+```
+$ sudo sed -i 's/^\(initrd.*\)/# \1/g' /usr/share/defaults/kata-containers/configuration.toml
+```
+
+If you do wish to use an initial ramdisk, see https://github.com/kata-containers/osbuilder.
```

Comment:
If you use an initial ramdisk, see https://github.com/kata-containers/osbuilder.